go.mod

```diff
@@ -73,7 +74,7 @@ require (
 	golang.org/x/crypto v0.0.0-20200414173820-0848c9571904
 	golang.org/x/net v0.0.0-20200324143707-d3edc9973b7e
 	golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d
-	google.golang.org/grpc v1.26.0
+	google.golang.org/grpc v1.28.1
```

Comment:
Why change grpc verstion to 1.28.1, and redirect version to 1.26 in the head?

Reply:
Because of the etcd bug,see this:
https://github.com/etcd-io/etcd/issues/11931

Reply:
Please try to donwgrade grpc to v1.26 which is more stable version.

Reply:
go.mod doesn't support inherit for `replace` option,  that cause third party developer have to update `go.mod` to have this `replace` change,  @foshantiger  could you please take more time to find a better solution here?

Reply:
Haven't find a solution better than `replace` yet, I will update the development doc as a notice for that.